api/envoy/extensions/rbac/custom_library_config/v3/custom_library.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.extensions.rbac.custom_library_config.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.rbac.custom_library_config.v3";
+option java_outer_classname = "CustomLibraryProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: RBAC CEL Custom Library]
+
+// [#extension: envoy.rbac.custom_library_config.custom_library]
+message CustomLibraryConfig {
```

Comment:
Please add a comment which describes the purpose of this message, and another for the field below.

Reply:
"CustomLibraryConfig" is a rather generic sounding name. Is "Custom Library" a term of art in this space?

Reply:
(related to the long comment I just added in custom_library.h) For both this comment and my one above, likely what we want is to rename this class and file for something like 'ExampleCustomCELVocabularyConfig' then in rbac.proto rename the field 'custom_library_config' to 'custom_cel_vocabulary_config' and there and elsewhere (in the Base factory) name the category of extensions something rbac-agnostic, like 'envoy.expr.custom_cel_vocabluary' or similar.

Reply:
Vocabulary or attribute. Whatever is most native to CEL.